data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+|LML101   |102455       |102456         |102454       |1            |
+|LML101   |102456       |               |             |1            |
+|LML102   |103767       |               |             |1            |
+|LML102   |103768       |               |             |2            |
+|LML102   |103769       |103767         |103768       |2            |
+|LML103   |108976       |108977         |108978       |1            |
+|LML103   |108977       |               |             |1            |
+|LML103   |108978       |               |             |2            |
+|LML104   |104666       |104667         |104668       |2            |
+|LML104   |104667       |-9             |-9           |1            |
+|LML104   |104668       |-9             |-9           |2            |
+
+
+| family_id            | Required, the family_id for the trio data                                           |
+| -------------------- | ----------------------------------------------------------------------------------- |
+| local\_participant\_id | Required, the local\_particpant\_id for each of the participants included in the trio |
+| paternal_id          | Optional, the local\_participant\_id for the father of the participant                |
+| maternal_id          | Optional, the local\_participant\_id for the mother of the participant                |
+| sex                  | Optional, the sex of the participant. 1 for male, 2 for female                      |
+
+### manifests/file_derivation.csv
+
+The **file_derivation.csv** manifest is only required for omics contributions with multiple filetypes generated through a bioinformatics pipeline or workflow.
+
+**file_derivation.csv** describes the relationships between files in a pipeline or workflow.
+
+| column                 | definition                                                      | type   |
+| ---------------------- | --------------------------------------------------------------- | ------ |
+| destination\_file\_group | The files in this file group is derived from source_file_group. | String |
+| source\_fil\e_group      | File group used to derive the destination_file_group.           | String |
+
+### manifests/env manifest
+
+For each script/notebook in _src/_, and each model in _models/_, there should be an env file (here env refers to a file named env with any extension, so env.yaml or env.txt, for example) that describes the environment in which it was created or run. Environment files should be named as follows: **descriptiveName_env** and placed in a folder called environments within the _configs/environment_ directory. Either individuals files or entire folders (whichever is the appropriate level) in scripts and notebooks within the _src/_ directory, or the _models/_ directory will need to be added to the env manifest file, matching them with their related environment file. See the see below for more infromation about this file:
```

Comment:
```suggestion
For each script/notebook in \_src/\_, and each model in \_models/\_, there should be an env file (here env refers to a file named env with any extension, so env.yaml or env.txt, for example) that describes the environment in which it was created or run. Environment files should be named as follows: **descriptiveName\_env** and placed in a folder called environments within the \_configs/environment\_ directory. Either individuals files or entire folders (whichever is the appropriate level) in scripts and notebooks within the \_src/\_ directory, or the \_models/\_ directory will need to be added to the env manifest file, matching them with their related environment file. See the see below for more information about this file:
``` 

Again, I think I'm interpreting what you're doing... that you want these underscores to appear, right?  If you have two or more in a single line of text, you **must** prefix them with the backslash to escape their normal markdown usage (italics)

Reply:
Thank you for catching these. Nicole and I thought we had corrected all of them, but more keep on showing up!